projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -484,7 +490,7 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      * ```
      */
     @Output()
-    public onSearchInput = new EventEmitter();
+    public onSearchInput = new EventEmitter<IComboSearchInputEventArgs>();
```

Comment:
Changing the output args would be a breaking change. This should be at least well documented in the changelog. Also, the PR should only be to 10.
I think that writing a migration for this would be pretty impossible.... The problem is that if anyone is doing custom filtering by using `combo.onSearchInput.subscribe(filterStr => filter(filterStr))` this change would straight up break it